crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -651,15 +651,14 @@ impl OlmMachine {
     /// Panics if a group session for the given room wasn't shared beforehand.
     ///
     /// [`share_group_session`]: Self::share_group_session
-    pub async fn encrypt(
```

Comment:
to make this a non-breaking change, shouldn't we leave a doc-hidden, marked as deprecated old `encrypt`-alias method, too?